### DISCUSSION TITLE
Content Review Request: Deep Learning for Gesture Recognition on ESP32-S3 from Training to Deployment

### DISCUSSION THREAD

**Discussion by BlakeHansen130** [espressif/developer-portal#344] [General]
Hello Espressif team,
I hope this message finds you well. I'd like to contribute an article to the Espressif Developer Portal about implementing deep learning-based gesture recognition on the ESP32-S3, covering the complete journey from training to deployment.
I have prepared the article in my forked repository, which you can find here:
https://github.com/BlakeHansen130/developer-portal/tree/deep-learning-gesture-esp32s3
I would greatly appreciate your review to ensure the content aligns with Espressif's writing conventions and quality standards. The article aims to help developers understand how to:

- Train a deep learning model for gesture recognition
- Optimize the model for embedded deployment
- Deploy and run the model on ESP32-S3

Please let me know if you need any additional information or if there are specific aspects you'd like me to modify. I'm open to any feedback to make this contribution as valuable as possible for the community.
Thank you for your time and consideration.
Best regards,
BlakeHansen130
[index.md](https://github.com/user-attachments/files/17965327/index.md)

**Comment by BlakeHansen130** [comment]
@espressif-bot

**Comment by f-hollow** [comment]
@BlakeHansen130 Hello, thank you very much for preparing the article!

Please feel free to create a pull request to this repo, and we will gladly do the review.

**Reply by BlakeHansen130** [reply]
Hello @f-hollow ,

Thank you very much for your feedback and support. I have now created Pull Request #350 for the article as requested.

Please feel free to review it, and let me know if there are any suggestions or feedback for further improvements. I am more than happy to make any changes needed to ensure the quality of the content.

Thank you again for your guidance and time.

Best regards,
BlakeHansen130
